### ISSUE TITLE
Not working with CommonJS

### ISSUE THREAD

**Issue by avolutions** [AlxndrVgt/colorado#5]
I'm using colorado 0.3.0 in my CommonJS project:

package.json:
```json
"type": "commonjs",
"dependencies": {
  "@avolutions/colorado": "^0.3.0"
}
```
index.js:
```js
const { Color, HtmlColors } = require('@avolutions/colorado');
```

When running the application with `node index.js` I receive the following error:

> node:internal/modules/package_json_reader:88
        throw error;
        ^
        SyntaxError: Error parsing C:\source\color-app\node_modules\@avolutions\colorado\dist\cjs\package.json: Expected property name or '}' in JSON at position 1
    at parse (<anonymous>)
    at Object.read (node:internal/modules/package_json_reader:75:16)
    at readPackage (node:internal/modules/cjs/loader:362:28)
    at readPackageScope (node:internal/modules/cjs/loader:391:19)
    at Module._extensions..js (node:internal/modules/cjs/loader:1261:17)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (C:\source\color-app\index.js:2:31) {
  path: 'C:\\source\\color-app\\node_modules\\@avolutions\\colorado\\dist\\cjs\\package.json'
}

**Comment by avolutions** [comment]
Reason is broken `package.json` for CommonJS (_/dist/cjs/package.json)_.
The file looks like this:
```json
{type: commonjs}
```
instead of:
```json
{ "type": "commonjs" }
```

**Comment by avolutions** [comment]
Solved in v0.3.1
